cmd/util/cmd/common/environment.go

```diff
@@ -6,6 +6,66 @@ import (
 	"github.com/onflow/flow-core-contracts/lib/go/templates"
 )
 
+var (
+	getInfoForProposedNodesScriptLocalnet = []byte(`
+		import FlowIDTableStaking from 0xf8d6e0586b0a20c7
```

Comment:
```suggestion
	getInfoForProposedNodesScript = []byte(`
		import FlowIDTableStaking from %s
```
Could we just inject the address and define the script once? (The addresses are also already defined in https://github.com/onflow/flow-go/blob/0d08168ebea6f854ad2e9ab7a58551a2646cf086/fvm/systemcontracts/system_contracts.go#L95)

Reply:
https://github.com/onflow/flow-go/pull/1988/commits/93b09c2b5b78314c38c0cb496c64378b5e740376